src/request-pipeline/index.ts

```diff
@@ -93,6 +96,7 @@ const stages = [
             await ctx.session.callRequestEventCallback(RequestEventNames.onConfigureResponse, rule, configureResponseEvent);
             await callOnResponseEventCallbackForFailedSameOriginCheck(ctx, rule, ConfigureResponseEventOptions.DEFAULT);
         });
+        logger.proxy('Proxy CORS check failed %s, respond 222', ctx.requestId);
```

Comment:
`Proxy CORS check failed %s, response 222` - if we got 222
`Proxy CORS check failed %s, responding 222` - if we are going to send 222